src/Validator/EditorValidatorV2.sol

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.13;
+
+import {IValidator} from "./IValidator.sol";
+
+contract EditorValidatorV2 is IValidator {
+    /// -----------------------------------------------------------------------
+    /// Errors
+    /// -----------------------------------------------------------------------
+
+    /// The ipfs hash provided hash a wrong length
+    error WrongIPFSLength();
+    /// The editor has reached the maximum edits allowed
+    error ExceededEditLimit();
+    /// The address provided does not correspond to any whitelisted editor
+    error EditorNotWhitelisted();
+
+    /// -----------------------------------------------------------------------
+    /// Storage variables
+    /// -----------------------------------------------------------------------
+
+    /// @dev fix before year 2106
+    mapping(address => uint32[5]) edits;
+    address owner;
+    mapping(address => bool) whitelistedAddresses;
+
+    /// -----------------------------------------------------------------------
+    /// Constructor
+    /// -----------------------------------------------------------------------
+    
+    constructor() {
+        owner = msg.sender;
+    }
+
+    /// -----------------------------------------------------------------------
+    /// Modifiers
+    /// -----------------------------------------------------------------------
+
+    /// @notice Check if the contract caller is the owner
+    modifier onlyOwner() {
+        require(msg.sender == owner, "Ownable: caller is not the owner.");
+        _;
+    }
+
+    /// -----------------------------------------------------------------------
+    /// External functions
+    /// -----------------------------------------------------------------------
+
+    /// @notice Add a new editor to the whitelist
+    /// @param editorAddress the address of the editor
+    function whitelistEditor(address editorAddress) external onlyOwner {
+        whitelistedAddresses[editorAddress] = true;
+    }
+
+    /// @notice Check if the editor is whitelisted
+    /// @param editorAddress the address of the editor
+    function isEditorWhitelisted(address editorAddress) external view returns (bool) {
+        return whitelistedAddresses[editorAddress]; 
+    }
+
+    /// @notice Review that an editor can post a wiki based in previous edits
+    /// @param _user The user to approve the module for
+    /// @param _ipfs The IPFS Hash
+    function validate(address _user, string calldata _ipfs) external returns (bool) {
+        if(!whitelistedAddresses[_user]) {
+            revert EditorNotWhitelisted();
+        }
+
+        uint32[5] memory userEdits = edits[_user];
```

Comment:
since users are whitelisted and curated we dont need all the limits part in this contract 👍🏻